src/state/slices/assetsSlice/selectors.ts

```diff
@@ -91,7 +91,7 @@ export const selectFeeAssetById = createCachedSelector(
   assetId: AssetId,
 ) => ReturnType<typeof getFeeAssetByAssetId>
 
-export const selectAssetsBySearchQuery = createDeepEqualOutputSelector(
+export const selectAssetsBySearchQuery = createCachedSelector(
```

Comment:
Could this yield wrongly cached results? i.e search while loading, get cached results, search again and get hit by one of two hard things in software engineering

Reply:
The cache gets invalidated by the key changing, but also by upstream deps changing, so we should theoretically be good

Reply:
Precisely, and for a given key, we currently would have different results as assets are partially dynamic - e.g searching for a UNI-V2 asset would initially yield the results of the UNI-V2 assets without meta, while after we fetch the meta for these assets and upsert them, a subsequent search would yield assets with full meta